src/algebra/euclidean_domain.lean

```diff
@@ -47,6 +47,9 @@ instance : has_mod R := ⟨euclidean_domain.remainder⟩
 theorem div_add_mod (a b : R) : b * (a / b) + a % b = a :=
 euclidean_domain.quotient_mul_add_remainder_eq _ _
 
+lemma mod_add_div (a b : R) : a % b + b * (a / b) = a :=
+by rw add_comm; exact div_add_mod _ _
```

Comment:
Could golf as:
```suggestion
(add_comm _ _).trans (div_add_mod _ _)
```
which eliminates the `;` which the style guide recommends against.